build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -51,8 +51,8 @@ jobs:
       downloadPath: '$(Build.SourcesDirectory)\Artifacts'
 
   # debugging - remove or comment out before completing PR      
-  #- script: |
-  #    dir /s $(Build.SourcesDirectory)
+  - script: |
```

Comment:
debugging? I'm fine if this stays but the comment should be updated if it'll be kept in.

Reply:
argh, I should have removed this.  I'll do this on the next PR (and it will be small so it gets in soon).